pkg/gitops/profile.go

```diff
@@ -70,10 +65,15 @@ func (p *Profile) Generate(ctx context.Context) error {
 	return nil
 }
 
-func (p *Profile) deleteClonedDirectory(path string) {
-	logger.Debug("deleting cloned directory %q", path)
-	if err := p.IO.RemoveAll(path); err != nil {
-		logger.Warning("unable to delete cloned directory %q", path)
+// DeleteClonedDirectory deletes the directory where the repository was cloned
+func (p *Profile) DeleteClonedDirectory() {
```

Comment:
I think that after the refactoring I suggested this is not needed. If not, I think its name should be consistent with the method added to the flux installer.